### ISSUE TITLE
ggplotly doesn't take an hoverinfo arguements

### ISSUE THREAD

**Issue by vikram-rawat** [ropensci/plotly#1255]
I have a ggplot2 which was created with several layers. The hoverinfo which ggplotly automatically picks up has duplicated and some redundant information. I want to change it.

look at the plot below and you can see what I mean by it.

![image](https://user-images.githubusercontent.com/22342596/39964467-095521f6-56a3-11e8-9d60-ba2ecd2129c8.png)

I want to be able to remove some of this information and add some of mine. I tried hoverinfo but it only works in plotly graph created through plotly.

When I use hoverinfo in ggplotly arguement it simply ignores it and moves on...

this is the code

```

(
fulldata %>% 
        ggplot(aes(date,cum,
           group=emp_code,
           text=paste('Name:',name)))+
    geom_line(aes(
        color=line_color,
        alpha=line_color))+
    scale_colour_manual(values=
            as.character(unique(fulldata$line_color))
                        )+
    scale_alpha_manual(values = c(black=.07,
                                  green=.15,
                                  orange=.4,
                                  red=.6))+
    geom_point(
        data = data_top_10,
        aes(date,
            cum,
            fill = I('steelblue')),
        size = 3,
        alpha = I(.4))+
        geom_point(data = data_last_date,
                       aes(date,
                           cum,
                           fill = I('royalblue')),
                       alpha = I(.12),
                    size=I(.7)) +
            
        geom_hline(
                yintercept = 50,
                color = 'red',
                size = I(.7),
                alpha = I(.7),
                linetype = 'dashed'
            ) + ylab('OverTime')
    ) %>% 
    ggplotly(hoverinfo='text+x+y') %>%
    hide_legend() %>%
    layout(plot_bgcolor = 'transparent',
           paper_bgcolor = 'transparent')

```

please tell me how should I do it. I want to use it in a shiny dashboard and I can't use this type of hoverinfo but hoverinfo is neccessary too...

**Comment by cpsievert** [comment]
Thanks!

As per our [contributing guidelines](https://github.com/plotly/plotly.js/blob/master/CONTRIBUTING.md), github issues are reserved for bug reports and feature requests only. So I'll have to close your ticket.

Please use our community forums at [community.plot.ly](http://community.plot.ly/) for posts of the likes.

**Comment by cpsievert** [comment]
To be a bit more helpful, see the section customizing tooltips -- http://blog.cpsievert.me/2018/01/30/learning-improving-ggplotly-geom-sf/

**Comment by vikram-rawat** [comment]
I am really very sorry to bug you here. I didn't find any hoverinfo in ggplotly. I googled it but found nothing that could help me and I tried to post the question in community.Rstudio.com and I didn't get any reply on it. 

I thought this feature might be missing. That's the reason I posted it here. But thanks for pointing out resources and I am really very sorry to bug you for no apparent reasons.
